tests/components/freebox/test_sensor.py

```diff
@@ -115,3 +115,21 @@ async def test_battery(
     assert hass.states.get("sensor.telecommande_niveau_de_batterie").state == "25"
     assert hass.states.get("sensor.ouverture_porte_niveau_de_batterie").state == "50"
     assert hass.states.get("sensor.detecteur_niveau_de_batterie").state == "75"
+
+
+async def test_get_hosts_list(
```

Comment:
We are fetching `lan.get_hosts_list` for the `device_tracker` platform, not `sensor`.

There is no `test_device_tracker.py` file now but you can add one.

So it should have 2 tests:
- 1 with tracked devices : classic Freebox config -> no bridge mode
- 1 with without tracked devices : Freebox in bridge mode

Reply:
Not resolved.

Move this test to `test_device_tracker.py`

And you have to count number of devices linked to the router (--> `router.devices`):
- 1 if in bridge mode : the router itself
- more than 1 if not

Reply:
Not resolved.

Move this test to `test_device_tracker.py`

And you have to count number of devices linked to the router (--> `router.devices`):
- 1 if in bridge mode : the router itself
- more than 1 if not